src/components/ProgressBar/ProgressBar.scss

```
@@ -44,7 +44,7 @@
   height: inherit;
   width: 0;
   will-change: width;
-  background-color: color('teal');
+  background-color: var(--p-highlight, color('teal'));
```

We want to avoid using the root variant, since it is unmodified from the original input.

```suggestion
  background-color: var(--p-highlight-border, color('teal'));
```

While it feels a bit strange using a border for this, it's the closest variant we have for this purpose.